plugins/config_manager.py

```diff
@@ -60,6 +60,20 @@ def configure(self):
         demands = self.cli.demands
         demands.available_repos = True
 
+        # if no argument was passed then error
+        if (not (self.opts.crepo != [] or
+                 self.opts.add_repo != [] or
+                 self.opts.save or
+                 self.opts.dump or
+                 self.opts.dump_variables or
+                 self.opts.set_disabled or
+                 self.opts.set_enabled) ):
+            self.cli.optparser.error(_("one of the arguments {} is required")
+                                     .format(' '.join([
+                                         "repo", "--save", "--add-repo",
+                                         "--dump", "--dump-variables",
+                                         "--enable", "--disable"])))
```

Comment:
I think that ``dnf config-manager fedora`` should also fail, because there is no action to do. So, can you remove the "repo" option(s) from the list?